policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/pension_income_amount/maximum_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Saskatchewan provides this maximum amount under the pension income credit.
+values:
+  2023-01-01: 1_000
+metadata:
+  unit: currency-CAD
+  label: Saskatchewan pension income maximun amount
+  reference:
+  - title: 2023 Saskatchewan Tax Credits Return
```

Comment:
Can you add a legal reference? If not please leave a comment saying it's unavailable.